packages/@react-spectrum/listbox/test/ListBox.test.js

```diff
@@ -489,6 +493,34 @@ describe('ListBox', function () {
       expect(document.activeElement).toBe(options[4]);
     });
 
+    it('supports the space character in a search', function () {
```

Comment:
if you use jest.useFakeTimers, you can step through time to verify that you can use space bar to select the option after the typeahead timeout has completed, i won't require that you write that test, just an interesting thing you can do

Reply:
Thanks, @snowystinger, good idea! I called it out in the test instructions, so I think I should include it as well as a test.

Reply:
Done.